source/features/github-bugs.css

```diff
@@ -40,16 +35,6 @@
 	display: none !important;
 }
 
-/* Avoid multi-column wrapping in PR lists on small displays #4405 */
```

Comment:
No longer reproducible on https://github.com/refined-github/refined-github/pulls?q=Drop+support+for+the+old+GitHub+layout+sort%3Aupdated-desc at 900px

<img width="861" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/183243816-c06f3855-00ac-431b-b82c-8b31e7caa804.png">